service/controller/chart/resource/release/update.go

```diff
@@ -243,10 +244,15 @@ func (r *Resource) newUpdateChange(ctx context.Context, obj, currentState, desir
 	}
 
 	if isReleaseModified(currentReleaseState, desiredReleaseState) {
-		r.logger.Debugf(ctx, "the %#q release has to be updated", desiredReleaseState.Name)
+		opt := cmp.FilterPath(func(p cmp.Path) bool {
+			return p.String() == "Values"
```

Comment:
This is to ignore `Values` at all, I tried to rename it to `values` to unable to exported but it brought broader changes that I didn't mean to.

Reply:
What happens if just the Values are changed?

Good to exclude the Values from the diff but we should log that somehow.

Reply:
I think md5 would be different then, we would show md5 hash into the difference. I think that's enough information we could show, WDYT?

Reply:
Yes totally. I forgot the MD5 hash will appear in the diff. 😅 

What about adding a comment to explain that? As its easy to miss when reading quickly.

Reply:
Good call, I added a comment as you suggested.